go/diff/apply_patch.go

```diff
@@ -0,0 +1,287 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package diff
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/attic-labs/noms/go/d"
+	"github.com/attic-labs/noms/go/types"
+)
+
+// ApplyPatch applies a Patch (list of diffs) to a graph. It makes the following
+// contract:
+//  Given 2 Noms graphs: a1 and a2:
+//    ApplyPatch(a1, Diff(a1, a2)) == a2
```

Comment:
It's a minor thing but I still `Apply` is best because this is in the `diff` package so it's very clear what you're applying.